### DISCUSSION TITLE
How to debug lerna project in vscode?

### DISCUSSION THREAD

**Discussion by raviSussol** [lerna/lerna#3574] [Q&A]
What's the correct configuration for debugging a lerna project in vscode? I have a start script `lerna run --scope @app-frontend/* --parallel start` which starts my react lerna project.
I tried to setup my `launch.json` file as below;
```
{
      "trace": true,
      "name": "Debug frontend",
      "type": "node",
      "request": "launch",
      "address": "localhost",
      "protocol": "inspector",
      "runtimeExecutable": "${workspaceRoot}/node_modules/.bin/lerna",
      "runtimeArgs": [
        "run",
        "--scope",
        "@app-frontend/*",
        "--parallel",
        "start"
      ]
}
```
That starts the project but doesn't hit any breakpoints in the app at all. What could be wrong in the above debug settings?
Any help would be appreciated. Thanks.

**Comment by ghiscoding** [comment]
hello, I have a [Troubleshooting](https://github.com/lerna-lite/lerna-lite/wiki/Troubleshooting) Wiki on how to debug in [Lerna-Lite](https://github.com/lerna-lite/lerna-lite), it's probably very similar for Lerna, you probably just need to adjust necessary bin location.
